helm_deploy/create-and-vary-a-licence-api/templates/app-jobs.yaml

```diff
@@ -0,0 +1,36 @@
+{{ $host := (index .Values "generic-service" "ingress" "host") }}
+{{- range .Values.jobs }}
```

Comment:
this will iterate over a range of job definitions in values.yaml if we decide to move over the other jobs to the api